<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>481</width>
    <height>54</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="pre_edit"/>
     </item>
     <item>
      <widget class="QLabel" name="placeholder_label">
       <property name="text">
        <string>Placeholder:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="arrow_label">
       <property name="text">
        <string>---&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="left_bracket">
       <property name="text">
        <string>{{</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="post_edit"/>
     </item>
     <item>
      <widget class="QLabel" name="right_bracket">
       <property name="text">
        <string>}}</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="remove_button"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
